src/raudio.c

```diff
@@ -397,10 +400,18 @@ void StopAudioBuffer(AudioBuffer *buffer);
 void PauseAudioBuffer(AudioBuffer *buffer);
 void ResumeAudioBuffer(AudioBuffer *buffer);
 void SetAudioBufferVolume(AudioBuffer *buffer, float volume);
+void SetAudioBufferPan(AudioBuffer* buffer, float pan);
 void SetAudioBufferPitch(AudioBuffer *buffer, float pitch);
 void TrackAudioBuffer(AudioBuffer *buffer);
 void UntrackAudioBuffer(AudioBuffer *buffer);
 
+
+// pan law
+static float FastSineApproximation(float x)
```

Comment:
I prefer to avoid adding a new function, code can be implemented directly where required.